src/main/java/beans/menus/EntityListResponse.java

```diff
@@ -230,7 +230,13 @@ private void processStyles(Detail detail) {
     }
 
     private void processActions(SessionWrapper session) {
-        Vector<Action> actions = session.getDetail(((EntityDatum) session.getNeededDatum()).getShortDetail()).getCustomActions(session.getEvaluationContext());
+        EntityDatum datum = (EntityDatum) session.getNeededDatum();
+        // HACKY! Uses the fact that case claim results will always be in a "result" instance
```

Comment:
@ctsims I couldn't find any cleaner way to determine whether we're on an EntityList as a result of a case claim action or a real case list than looking at the instance name. Any brighter ideas appreciated